brukere.txt

```diff
@@ -1 +1,2 @@
 Ole Martin Knurvik
+Philip Dahlstrøm
```

Comment:
Pretty controversial change, are you sure about this?

Reply:
I disagree @mariussd. I think it is a good enhancement. Good job, Philip!